src/frontend/src/pages/ProjectDetailPage/ProjectEdit/ProjectEditContainer.tsx

```diff
@@ -84,15 +101,18 @@ const ProjectEditContainer: React.FC<ProjectEditContainerProps> = ({ project, ex
     defaultValues: {
       name,
       budget,
-      slideDeckLink,
-      bomLink,
-      taskListLink,
-      googleDriveFolderLink: gDriveLink,
       summary,
       crId: query.get('crId') || '',
       rules: project.rules.map((rule) => {
         return { rule };
       }),
+      links: projectLinks.map((link) => {
```

Comment:
also this map is unnecessary, you're just making a new object that's the same as the old one then returning it

Reply:
no this is required for useFieldArray

Reply:
I think....

Reply:
what is it doing then?